pages/_en-US/ds-index/hardmod.md

```diff
@@ -11,4 +11,90 @@ Hardmodding is when you physically solder the motherboard of the Nintendo DSi to
+  - SD to microSD adapter
+  - An SD card reader able to read eMMC chips running in single data-line mode. [Here's a known working one](https://www.amazon.com/gp/product/B006T9B6R2)
+  - [Win32DiskImager](https://sourceforge.net/projects/win32diskimager/) and [HxD](https://mh-nexus.de/en/downloads.php?product=HxD20) if on Windows
+  - A valid DSi backup of the DSi you are hardmodding
+  - To be comfortable soldering to pads that are as small as 0.5mm in diameter
+
+#### DSi side A pinouts
+![DSi side A pinouts](/assets/images/dsi-hardmod/side_a.jpg)
+#### DSi side B pinouts
+![DSi side B pinouts](/assets/images/dsi-hardmod/side_b.jpg)
+#### DSi XL side B pinouts
+![DSi XL side B pinouts](/assets/images/dsi-hardmod/dsi_xl_side_b.png)
+
+- You will need to solder the points on the motherboard to the microSD card adapter
+  - CMD to pin 2
+  - GND to pins 3 and 6
+  - CLK to pin 5
+  - DAT0 to pin 7
+
+#### Soldered microSD adapter example
+![microSD example](/assets/images/dsi-harmod/sd.jpg)
+
+- Insert the microSD card adapter to your computer
+  - **WARNING** - Windows will ask you to format it: **DO NOT** format it - this might cause irrepairable damage
+
+#### Windows instructions (skip for macOS / Linux)
+
+1. Open Win32DiskImager
+2. Click the folder icon and browse to your desktop. In the text box, write NAND_0.bin. When selecting type, choose All types *.* 
+3. Choose the device that is the DSi and click read. 
+4. Once it is done, click the folder icon,change the name to NAND_1.bin and read again
+5. Open HxD and drag both files into the editor. Go to the top bar, click "Analysis",click "File compare" from the drop down menu, then click "Compare".
+6. Choose to compare both files then click OK when done
+  - If it says "The chosen files are identical.", you are good to go to the next section
+  - If it doesn't say that, and both NANDs aren't around 240MB, dump NAND_1/NAND_0 again
+7. Open Win32DiskImager, click the folder icon and choose your existing NAND backup you created before.
+8. Flash it with the `Write` button.
+9. Unplug the SD card adapter and attempt to turn on the DSi
+
+#### Linux / macOS instructions (skip if on Windows)
+
+
+1. Work out where the SD card is mounted
+  - Linux:
+    1. Unplug the SD card adapter
+    2. Run `lbslk` on a terminal
+    3. Plug in the SD card adapater
+    4. Run `lbslk` again
+    5. This time a new device should appear
+    6. It may be called `/dev/sdb`, make sure you note down what it is called on *your* terminal
+
+  - macOS:
+    1. Unplug the SD card adapter
+    2. Run `df` on a terminal
+    3. Plug in the SD card adapter
+    4. Run `df` again
+    5. A new device should appear in the list
+    6. It may be called `/dev/disk1s1`, make sure you note down what it is called on *your* terminal/
```

Comment:
Instead of makking this a new number step, try making this a sub-bullet of "A new device should appear in the list"